esi_leap/manager/service.py

```diff
@@ -63,28 +60,29 @@ def stop(self):
 
     def _fulfill_contracts(self):
         LOG.info("Checking for contracts to fulfill")
-        contracts = contract.Contract.get_all_by_status(
-            self._context, statuses.CREATED)
+        contracts = contract.Contract.get_all(
+            self._context, {'status': [statuses.ACTIVE, statuses.CREATED]})
```

Comment:
If a contract is active, I don't think we need to fulfill it again.

Reply:
fixed